package.json

```diff
@@ -84,6 +84,7 @@
     "handlebars": "^4.7.8",
     "input-otp": "^1.2.4",
     "js-cookie": "^3.0.5",
+    "json2csv": "^6.0.0-alpha.2",
```

Comment:
@aryany9 this package is abandoned. please remove it and change the approach
https://www.npmjs.com/package/json2csv